gui.py

```diff
@@ -270,6 +276,27 @@ def get_local_config(self):
         devices, shifts = self.get_device_details()
         return config_template.format(self.textbox_erpnext_api_key.text(), self.textbox_erpnext_api_secret.text(), self.textbox_erpnext_url.text(), self.textbox_pull_frequency.text(), formated_date, json.dumps(devices), json.dumps(shifts))
 
+    def get_running_status(self):
+        running_status = []
+        found = False
+        with open('/'.join([config.LOGS_DIRECTORY])+'/logs.log', 'r') as f:
+            for line in f:
+                logdate = datetime.datetime.strptime(line.split(',')[0], '%Y-%m-%d %H:%M:%S')
+                if logdate >= datetime.datetime.strptime(self.service_start_time.text().split('.')[0] , '%Y-%m-%d %H:%M:%S'):
+                    running_status.append(line)
+        with open('/'.join([config.LOGS_DIRECTORY])+'/error.log', 'r') as f:
+            for line in f:
+                start_date = datetime.datetime.strptime(self.service_start_time.text().split('.')[0] , '%Y-%m-%d %H:%M:%S')
+                if start_date.strftime('%Y-%m-%d') in line:
+                    logdate = datetime.datetime.strptime(line.split(',')[0], '%Y-%m-%d %H:%M:%S')
+                    if logdate >= start_date:
+                        found = True
+                if found:
+                    running_status.append(line)

```

Comment:
also, error.log file will contain python error tracebacks... and these lines will not have a timestamp... if you find a line that has a logtime > start time, can you include all the lines after that?

Reply:
changes updated. kindly, check this commit https://github.com/frappe/biometric-attendance-sync-tool/pull/17/commits/9f22725a91374689d0251f3ef5aeac1a6df74e2b